src/views/catalog/CreateFromInstanceTypes/components/AddBootableVolumeButton/AddBootableVolumeButton.tsx

```diff
@@ -24,12 +23,16 @@ const AddBootableVolumeButton: FC<AddBootableVolumeButtonProps> = ({ buttonVaria
     namespace: isUpstream ? KUBEVIRT_OS_IMAGES_NS : OPENSHIFT_OS_IMAGES_NS,
     group: DataSourceModel.apiGroup,
   });
-  const { instanceTypesAndPreferencesData } = useInstanceTypeVMStore();
+  const { instanceTypesAndPreferencesData, onSelectVolume } = useInstanceTypeVMStore();
```

Comment:
can you please have the same name for onSelectVolume and onCreateVolume? I would call both onSelectCreatedVolume as suggested above

Reply:
renamed the `useInstanceTypeVMStore` `onSelectVolume` method as `onSelectCreatedVolume`